voter-dapp/src/ActiveRequests.js

```diff
@@ -13,9 +15,14 @@ import { decryptMessage, deriveKeyPairFromSignatureMetamask } from "./common/Cry
 const { getKeyGenMessage } = require("./common/EncryptionHelper.js");
 
 function ActiveRequests() {
-  const { drizzle, useCacheCall, useCacheEvents } = drizzleReactHooks.useDrizzle();
+  const { drizzle, useCacheCall, useCacheEvents, useCacheSend } = drizzleReactHooks.useDrizzle();
   const { web3 } = drizzle;
 
+  const [checkboxesChecked, setCheckboxesChecked] = useState({});
+  const check = (index, event) => {
+    setCheckboxesChecked(old => ({ ...old, [index]: event.target.checked }));
```

Comment:
After thinking about this a bit more, I think this is probably the easiest way to build the "array".